var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -148,6 +148,13 @@ class IntelParallelStudio(IntelPackage):
     conflicts('+vtune',     when='@composer.0:composer.9999')
 
     @property
+    def intel_prefix(self):
+        if self.version[0] == "composer" and self.version[1] == 2015:
```

Comment:
Is `self.version[1]` equal to `2015` or `'2015'`? Can't remember whether we cast to int or not.